packages/boot/test/test-boot-config.js

```diff
@@ -4,18 +4,16 @@ import '@agoric/swingset-liveslots/tools/prepare-test-env.js';
 import anyTest from 'ava';
 import { spawn as ambientSpawn } from 'child_process';
 import { promises as fsPromises } from 'fs';
-import { resolve as importMetaResolve } from 'import-meta-resolve';
 import path from 'path';
 
 import { extractCoreProposalBundles } from '@agoric/deploy-script-support/src/extract-proposal.js';
+import { pkgAbsPath } from '@agoric/swingset-vat/tools/paths.js';
 import { mustMatch } from '@agoric/store';
 import { loadSwingsetConfigFile, shape as ssShape } from '@agoric/swingset-vat';
 import { provideBundleCache } from '@agoric/swingset-vat/tools/bundleTool.js';
 
 const importConfig = configName =>
-  importMetaResolve(`@agoric/vm-config/${configName}`, import.meta.url).then(
-    u => new URL(u).pathname,
-  );
+  pkgAbsPath(`@agoric/vm-config/${configName}`);
```

Comment:
The correctness of this expression relies on the incidental detail that `@agoric/vm-config` got installed somewhere accessible to `@agoric/swingset-vat`. Your `pnpm` experiment presumably revealed that this is not a reliable incidental detail unless `@agoric/vm-config` also explicitly depends upon `@agoric/swingset-vat`, and even then, is not guaranteed unless both of these packages depend on compatible versions.

Reply:
I suspect that peerDependencies will save the day for vm-config.